src/core/a-register-element.js

```diff
@@ -58,6 +58,14 @@ module.exports.registerElement = function (tagName, obj) {
     newObj = {prototype: Object.create(proto, newObj)};
   }
 
+  // Give all functions their proper name.
+  for (var propName of Object.getOwnPropertyNames(newObj.prototype)) {
+    var propVal = newObj.prototype[propName];
+    if (propVal && typeof propVal === 'function') {
```

Comment:
Is checking for `if ( typeof propVal === 'function')` sufficient?

Reply:
Ah, yes. Will fix.